IPython/html/static/edit/js/main.js

```diff
@@ -29,22 +29,29 @@ require([
     ){
     page = new page.Page();
 
-    var base_url = utils.get_body_data('baseUrl');
-    var file_path = utils.get_body_data('filePath');
-    contents = new contents.Contents({base_url: base_url});
-    var config = new configmod.ConfigSection('edit', {base_url: base_url});
+    var common_options = {
+        ws_url : utils.get_body_data("wsUrl"),
+        base_url : utils.get_body_data("baseUrl"),
+        file_path : utils.get_body_data('filePath'),
+        notebook_path : utils.get_body_data("notebookPath"),
+        notebook_name : utils.get_body_data('notebookName')
+    };
+
+    // var base_url = utils.get_body_data('baseUrl');
+    // var file_path = utils.get_body_data('filePath');
+    var config = new configmod.ConfigSection('edit', common_options);
```

Comment:
I'm not sure this is the direction we should be going in - common_options is a convenient thing, but it's better to pass constructors only the set of options they use where that's practical. So I'd like to use common_options less, not more.